src/restapi/fDisconnect.ts

```diff
@@ -26,8 +33,13 @@ export async function fDisconnect(req: Request, res: Response): Promise<void> {
     }
 
     const zdClient = await newZDClient(zdOptions);
-    const oauthReq = zdClient.oauthtokens.list();
-    const tokens: ZDTokensResponse = await tryPromiseWithMessage(oauthReq, 'failed to get oauth tokens');
+    let tokens: ZDTokensResponse;
+    try {
+        tokens = await zdClient.oauthtokens.list();
```

Comment:
I would love to see `zendesk.ts` to be properly typed, but we can deffer that to a different PR.

Maybe it's worth to submit a PR to https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node-zendesk/index.d.ts and ditch the local file.

Reply:
Ticket created. https://mattermost.atlassian.net/browse/MM-35874

I have had a couple of PRs accepted and merged for the ZD client repo, but haven't started adding PRs to the ZD types repo.